components/external_server/src/routes/metric.py

```diff
@@ -172,5 +173,6 @@ def create_issue_description(metric, subject, report) -> str:
     metric_url = dict(bottle.request.json)["metric_url"]
     return (
         f"Metric '[{metric.name}|{metric_url}]' of subject '{subject.name}' "
-        f"in Quality-time report '{report.name}' needs attention."
+        f"in Quality-time report '{report.name}' needs attention.\n\n"
+        f"Why address '{metric.name}'? {DATA_MODEL.metrics[metric.type()].rationale}"
```

Comment:
Are all current rationales suitable for including? Perhaps something to put in a hover/tooltip?

Reply:
This is text that's inserted into Jira issues. Not sure you can make a tooltip in a Jira issue?

Reply:
I checked, but I don't see an option to create a tooltip in the description of a Jira issue. All metrics have a rationale, so at least there will be some text. If the text is not suitable, the user can always delete it when editing the issue description in Jira. And if the rationale is never useful to have in the issue, I'm sure they'll let us know :-)